src/Mono.Android/Android.Util/Log.cs

```diff
@@ -1,13 +1,19 @@
 using System;
+using System.Globalization;
 using Android.Runtime;
 
 namespace Android.Util {
 
 	public partial class Log {
 
+		/// <summary>
+		/// IFormatProvider passed to any underlying string.Format() calls. Defaults to System.Globalization.CultureInfo.CurrentCulture.
+		/// </summary>
+		public static IFormatProvider FormatProvider { get; set; } = CultureInfo.CurrentCulture;
```

Comment:
Doh, one last change: this should be wrapped in a `#if ANDROID_34` block, as we shouldn't be updating API-33!

Reply:
Should probably just change visibility?

```csharp
#if ANDROID_34
    public
#endif
    static IFormatProvider FormatProvider { get; set; } = CultureInfo.CurrentCulture;
```